src/app/lib/utilities/preprocessor/rules/cpsAssetPage/addSummaryBlock/index.js

```diff
@@ -0,0 +1,43 @@
+import { singleTextBlock } from '#app/models/blocks';
+import pathOr from 'ramda/src/pathOr';
+import deepClone from 'ramda/src/clone';
+import path from 'ramda/src/path';
+
+const getSummary = json => {
+  const summary = pathOr(null, ['promo', 'summary'], json);
+
+  if (!summary) {
+    return json;
+  }
+
+  return summary;
+};
+
+const addSummaryBlocks = originalJson => {
```

Comment:
Consider early return to ensure only ran on applicable assets, at the moment PGL